pkg/sink/codec/encoder_group.go

```diff
@@ -70,15 +76,40 @@ func NewEncoderGroup(builder RowEventEncoderBuilder,
 	for i := 0; i < concurrency; i++ {
 		inputCh[i] = make(chan *future, defaultInputChanSize)
 	}
+	outCh := make(chan *future, defaultInputChanSize*concurrency)
+
+	var bootstrapWorker *bootstrapWorker
+	protocol := util.GetOrZero(cfg.Protocol)
+	if protocol == config.ProtocolSimple.String() {
+		log.Info("Sending bootstrap event is enable for simple protocol, creating bootstrap worker...",
+			zap.Stringer("changefeed", changefeedID))
+		sendBootstrapIntervalInSec := util.GetOrZero(cfg.SendBootstrapIntervalInSec)
+		if sendBootstrapIntervalInSec <= 0 {
+			sendBootstrapIntervalInSec = config.DefaultSendBootstrapIntervalInSec
+		}
+		msgCount := util.GetOrZero(cfg.SendBootstrapInMsgCount)
+		if msgCount <= 0 {
+			msgCount = config.DefaultSendBootstrapInMsgCount
+		}
+		interval := time.Duration(sendBootstrapIntervalInSec) * time.Second
+		bootstrapWorker = newBootstrapWorker(
```

Comment:
when creates the `bootstrapWorker`, `sendBootstrapIntervalInSec` and  should be passed as variables.

Reply:
The caller should not know too much about the `bootstrapWorker`.